src/modules/whats-new-modal/whats-new-steps.tsx

```diff
@@ -0,0 +1,54 @@
+import { useI18n } from '@wordpress/react-i18n';
+import { useState } from 'react';
+import Button from 'src/components/button';
+import IllustrationSvg from './illustration-svg';
+import WhatsNewModal from './whats-new-modal';
+
+export function WhatsNewButton( { className }: { className?: string } ) {
+	const { __ } = useI18n();
+	const [ isModalOpen, setIsModalOpen ] = useState( false );
+
+	// Example steps for the modal
+	const steps = [
```

Comment:
I think we could consider moving the steps into a separate file in the `whats-new-modal` module.

Maybe we could have another directory inside `whats-new-modal`  that would hold all the items content - including the image resources. Just to separate logic of the modal from its contents.